official-allow_functionality.json

```diff
@@ -229,8 +229,43 @@
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.s3-us-west-2.amazonaws.com"}},
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.cloudflare.com"}},
 		{"o":{"h":"*.hansje.net"},"d":{"h":"*.cargocollective.com"}},
-		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}}
+		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}},
+		{"o":{"h":"*.eurobuch.com"},"d":{"h":"*.lehmanns.de"}},
+		{"o":{"h":"*.emp.de"},"d":{"h":"*.acfrg.com"}},
+		{"o":{"h":"*.microsoft.com"},"d":{"h":"*.s-msft.com"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.trackersimulator.org"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.eviltracker.net"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.do-not-tracker.org"}},
+		{"o":{"h":"*.shortfil.ms"},"d":{"h":"*.eay.cc"}},
```

Comment:
I will make the required changes. Yes, another solution (it was suggested somewhere) would be separate subscriptions that users would be able to enable separately (eg _Multimedia content_ subscription would whitelist `vimeo.com vimeocdn.com youtube.com ytimg.com dailymotion.com dmcdn.com soundcloud.com sndcdn.com`)

@myrdd I'd like some feedback on this, currently I don't think RP would even allow this; IIRC the available subscriptions are somehow hardcoded. Do you agree with the general idea?